kubectl-fdb/cmd/root.go

```diff
@@ -80,11 +81,40 @@ func NewRootCmd(streams genericclioptions.IOStreams) *cobra.Command {
 		newDeprecationCmd(streams),
 		newFixCoordinatorIPsCmd(streams),
 		newGetCmd(streams),
+		newBuggifyCmd(streams),
 	)
 
 	return cmd
 }
 
+func getProcessGroupIDsFromPod(kubeClient client.Client, clusterName string, podNames []string, namespace string) ([]string, error) {
```

Comment:
This doesn't belong in the `root.go` we should move this into the `k8s_client.go`.